content/src/service.ts

```diff
@@ -1,14 +1,14 @@
 import { Lifecycle } from '@well-known-components/interfaces'
 import { EnvironmentConfig } from './Environment'
-import { migrateContentFolderStructure } from './migrations/ContentFolderMigrationManager'
+import { cleanSnapshots } from './logic/snapshot-cleaner'
 import { bootstrapFromSnapshots } from './service/synchronization/bootstrapFromSnapshots'
 import { AppComponents } from './types'
 
 // this function wires the business logic (adapters & controllers) with the components (ports)
 export async function main(program: Lifecycle.EntryPointParameters<AppComponents>): Promise<void> {
   const { components, startComponents } = program
 
-  await migrateContentFolderStructure(components)
```

Comment:
I don't think this is a good moment to remove the `migrateContentFolderStructure(components)`, it still migrates hundreds of files every time the service starts. Do we know why it happens?

Reply:
I didn't know. Will get it back and investigate why.

Reply:
I agree that we should remove that in another PR, anyway I think that the logger says that you are migrating files but there's nothing to do. We'll have to validate that.